k8s/namespaces/jenkins/patches/ptlsbox/cluster-00/jenkins.yaml

```diff
@@ -9,8 +9,12 @@ spec:
     controller:
       ingress:
         hostName: sds-sandbox-build.platform.hmcts.net
+        annotations:
+          traefik.ingress.kubernetes.io/router.tls: "true"
```

Comment:
this is already set in the default file no need to override it in the env specific one
https://github.com/hmcts/sds-flux-config/blob/master/k8s/namespaces/jenkins/jenkins.yaml#L265

```suggestion
```